packages/gravity-ui-web/src/sass/05-components/03-organisms/_list-cards-basic.scss

```diff
@@ -0,0 +1,46 @@
+.grav-c-list-cards-basic {
+  $breakpoint-1: gravy-breakpoint(medium);
+  $breakpoint-2: gravy-breakpoint(large);
+  padding: 0;
+  list-style-type: none;
+
+  @media all and (min-width: $breakpoint-1) {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  > li + li {
+    margin-top: $grav-sp-xl;
+  }
+
+
+  @media all and (min-width: $breakpoint-1) {
+    > {
+      // 2 col layout
+      li {
+        display: flex;
+        flex: 50%;
+        padding: 0 $grav-sp-l;
```

Comment:
Hmm... I assume this horizontal padding is to create the gutter between adjacent cards. Trouble is, it also creates a visible inset along the left and right edges of the list, when in 2 coloumn layout.

I suggest removing this padding and instead changing the flex basis above to something like `calc( (100% - #{$grav-sp-xl}) / 2);`.